### PR TITLE
Fixed split depth of cellbox to be loaded from config, amended circle…

### PR DESCRIPTION
**Pull request by hjabbot**
TL;DR:
Fixed split depth from being hard coded to now being read in the config.
Adjusted the abstract shape circle regression test to reflect this change. 

image

All regression tests pass

------

Minimum datapoints required before homogeneity condition returns 'MIN' was hardcoded in `cellbox.py` to be <4. Default passed into parameters is usually 5. This meant that any cellbox with exactly 4 datapoints would not be caught by the MIN check.

Fixing this caused the regression tests to fail.

[Broken regression tests](https://github.com/antarctica/PolarRoute/files/10698905/pytest_DUMP.txt)

In all the regression tests, only `Abstract_Environmental_Meshes/circle_n201_r2_cy-62.5_cx-60.0_mesh.json` included this situation where only 4 datapoints exist in a cellbox, so a new output was generated for it and included with the push.

[Amended regression tests](https://github.com/antarctica/PolarRoute/files/10698997/pytest_DUMP_fixed.txt)

All regression tests pass now.

### CODE REVIEW

**Review by SamuelHall700 — APPROVED** [review]
A good find of a pretty well hidden bug! It only caused a change in one of our tests cases when it was fixed it was that well hidden.
